common/src/main/java/com/wynntils/models/activities/markers/ActivityMarkerProvider.java

```diff
@@ -35,7 +36,12 @@ public void setSpawnLocation(Location spawnLocation) {
                                 Texture.QUESTS_SCROLL_ICON,
                                 CommonColors.WHITE,
                                 CommonColors.WHITE,
-                                CommonColors.WHITE),
+                                CommonColors.WHITE,
+                                Managers.Feature.getFeatureInstance(ContentTrackerFeature.class)
+                                                .showAdditionalTextInWorld
+                                                .get()
+                                        ? ""
```

Comment:
Can you comment in the code what this empty string is supposed to fix? So whenever we get back to this, we know what happened here.

Reply:
Alright, do I tag it with the FIXME?